app/presenters/email_alert_presenter.rb

```diff
@@ -24,6 +25,19 @@ def body
     end
   end
 
+  def tags
+    { format: document.publishing_api_document_type }.deep_merge(metadata)
+  end
+
+  def metadata
```

Comment:
IMHO, if it has to be dynamic, then this is fine, but if it could be static, an explicit list of fields would read better. @elliotcm any thoughts? I don't really have enough context here.